dynawo/sources/Models/Modelica/Dynawo/Electrical/Machines/SignalN/GeneratorPV_INIT.mo

```diff
@@ -0,0 +1,41 @@
+within Dynawo.Electrical.Machines.SignalN;
+
+/*
+* Copyright (c) 2015-2020, RTE (http://www.rte-france.com)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source time domain simulation tool for power systems.
+*/
+
+model GeneratorPV_INIT "Initialisation model for generator PV based on SignalN for the frequency handling"
+
+  import Dynawo;
+  import Dynawo.Electrical.Machines;
+
+  extends Machines.BaseClasses_INIT.BaseGeneratorSimplified_INIT;
+  extends AdditionalIcons.Init;
+
+    parameter Types.ActivePowerPu PMin "Minimum active power in MW";
+    parameter Types.ActivePowerPu PMax "Maximum active power in MW";
+    parameter Types.ReactivePowerPu QMin  "Minimum reactive power in MVAR";
+    parameter Types.ReactivePowerPu QMax "Maximum reactive power in MVAR";
+
+    Types.ActivePowerPu PMinPu "Minimum active power in p.u (base SnRef)";
+    Types.ActivePowerPu PMaxPu "Maximum active power in p.u (base SnRef)";
+    Types.ReactivePowerPu QMinPu  "Minimum reactive power in p.u (base SnRef)";
+    Types.ReactivePowerPu QMaxPu  "Maximum reactive power in p.u (base SnRef)";
```

Comment:
Simply do final parameters instead of equations.

Reply:
Is it possible to do so ? I thought the parameters calculated in the INIT model should be variables in this INIT model.